packages/amplify-graphql-transformer-core/src/utils/authType.ts

```diff
@@ -76,6 +76,7 @@ export const adoptAuthMode = (
       return {
         authorizationType: authType,
         lambdaAuthorizerConfig: {
+          lambdaArn: entry.lambdaAuthorizerConfig!.lambdaArn,
```

Comment:
Is this `!` intended since this is an optional property?

Reply:
If this is invoked after you construct it here https://github.com/aws-amplify/amplify-category-api/pull/2007/files#diff-d7d5f095c801bacf0479e8cd421efbcb56d9280b9e3a7eb1d254999c6e472f80R60, it should be fine.

Reply:
Yeah, there's probably a way to get the types to winnow correctly, but since the authorizationType has matched this is expected to exist.